src/portal/lib/src/tag/tag.component.ts

```diff
@@ -566,6 +573,25 @@ export class TagComponent implements OnInit, AfterViewInit {
     }
   }
 
+  retag(tags: Tag[]) {
+    this.retagDialogOpened = true;
+    this.retagSrcImage = this.repoName + ":" + tags[0].digest;
+  }
```

Comment:
if tags is empty array it will have undefined error if you use tags[0]

Reply:
you are right